crates/forge/tests/it/test_helpers.rs

```diff
@@ -612,10 +612,10 @@ pub async fn run_zk_script_test(
     let content = foundry_common::fs::read_to_string(run_latest).unwrap();
 
     let json: serde_json::Value = serde_json::from_str(&content).unwrap();
-    assert_eq!(
-        json["transactions"].as_array().expect("broadcastable txs").len(),
-        expected_broadcastable_txs
-    );
+    let txs = json["transactions"].as_array().expect("broadcastable txs");
+    tracing::debug!(?txs, "executed script");
```

Comment:
Do we need the log in tests?

Reply:
It's going to not be there most of the time, if not captured even by the test runner, but this way we can still have the information on what are the txs when stuff fails. I did add it for my own debugging initially but thought it could be useful overall